portality/formcontext/choices.py

```diff
@@ -100,10 +100,15 @@ class Choices(object):
     ]
 
     _application_status_editor = _application_status_base + [
-        ('on hold', 'On Hold'),
         ('ready', 'Ready'),
     ]
 
+    _application_status_publisher = [
```

Comment:
Hmm, giving the publishers a list of allowed statuses doesn't do anything, it turns out - the publisher doesn't see the application status on the form, as I had imagined from the description of #801. I'll simplify this in another commit as it's misleading just now.